### DISCUSSION TITLE
2. Ke2 annotated as blunder

### DISCUSSION THREAD

**Discussion by HukuToc2288** [official-stockfish/Stockfish#4774] [Ideas]
### Describe the issue

I recently knew about opening called [Bongcloud attack](https://en.wikipedia.org/wiki/Bongcloud_Attack), which starts as 1. e4 e5 2. Ke2. The idea of the opening is to develop a king and prepare for an early endgame. Some famous grandmasters like Hikaru Nakamura and former world champion Magnus Carlsen sometimes plays this opening against high-tier opponents and winning. On 15 March 2021 at Magnus Carlsen Invitational tournament Carlsen and Nakamura both played this attack and the game was quickly drawn after realizing there's no possible winning move against the bongcloud.

So, I tried to use this attack in my games. I still doesn't have much success with this opening, probably because I don't qualifying enough for this attack now. After most games I making game analysis to find what I did wrong and whenever I do it, stockfish annotating Ke2 as blunder (??). I understand that this attack may not be the best move leading to risky game and can be counterplayed by high-ranked players, but still every chess player using them own strategy and some players prefer risky and aggressive game with many gambits and countergambits over quiet game. So I want developers to consider make Ke2 not blunder.

### Expected behavior
Ke2 not annotated as blunder (??). I know stockfish can't annotate brilliant moves (!!), but this shouldn't be annotated at all or inaccuracy (?!) at least.

### Steps to reproduce
1. Start the game or the analysis
2. Play 1. e4 e5 2. Ke2

### Operating system
All (Find this bug on lichess.org)

### Stockfish version
14+

**Comment by Disservin** [comment]
These annotations are not coming from stockfish but from lichess

**Reply by HukuToc2288** [reply]
Oh, sorry, didn't knew that. I thought stockfish can both play and annotate games. Okay, I will close issue, sorry for disturbing you

**Comment by LazyFrankB** [comment]
I'm not developer, but 2. Ke2 indeed is a blunder. Humans play such moves to kick opponent out of preparation, but that doesn't work against engines.

**Reply by vdbergh** [reply]
As the OP said, it is about developing the king :smile:
